model/task/db.go

```diff
@@ -188,17 +188,17 @@ var (
 		},
 	}
 
-	updateDisplayTasksAndTasksExpression = bson.M{
-		"$set": bson.M{
-			CanResetKey: true,
-		},
-		"$unset": bson.M{
-			AbortedKey:              "",
-			AbortInfoKey:            "",
-			OverrideDependenciesKey: "",
+	updateDisplayTasksAndTasksSet = bson.M{"$set": bson.M{
+		CanResetKey: true,
+		ExecutionKey: bson.M{
```

Comment:
mostly out of curiosity, does inc not exist anymore? or does this have a behavior you prefer?

Reply:
[`$inc`](https://www.mongodb.com/docs/manual/reference/operator/update/inc/) is only an update operator, not an agg pipeline stage! There were a few cases like this to work around.